vcr/config.py

```diff
@@ -163,7 +165,12 @@ def _build_before_record_response(self, options):
         before_record_response = options.get(
             'before_record_response', self.before_record_response
         )
+        decode_compressed_response = options.get(
+            'decode_compressed_response', self.decode_compressed_response
+        )
         filter_functions = []
+        if decode_compressed_response:
+            filter_functions.append(filters.decode_response)
```

Comment:
That's the way it works now: decode_response (if the option is set) is added as the first filter, then everything passed in from `before_record_response` is appended to the end of the filter_functions list.